specification/caas-published-cohort-definitions.yaml

```diff
@@ -27,6 +27,188 @@ info:
     To be completed.
     ## Onboarding
     To be completed.
+
+    ### Sandbox testing
+    Our [sandbox environment](https://digital.nhs.uk/developer/guides-and-documentation/testing#sandbox-testing):
+    * is for early developer testing
+    * only covers a limited set of scenarios
+    * is stateless, so does not actually persist any updates
+    * is open access, so does not allow you to test authorisation
+
+    For details of sandbox test scenarios, or to try out the sandbox using our 'Try this API' feature, see the documentation for each endpoint.
+
+    ### You can try out the sandbox importing our Postman collection:
+    [published-cohort-definitions.postman_collection.json](../tests/postman-collection/published-cohort-definitions.postman_collection.json)
+    
+    ```json
+    {
+      "info": {
+        "_postman_id": "5ce2b578-3079-4ea9-a33e-06ed64a0ccb2",
+        "name": "published-cohort-definitions",
+        "schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+        "_exporter_id": "18640524"
+      },
+      "item": [
+        {
+          "name": "PublishedCohortLibraryGetBySlugName - SANDBOX",
+          "request": {
+            "method": "POST",
+            "header": [],
+            "body": {
+              "mode": "graphql",
+              "graphql": {
+                "query": "query PublishedCohortLibraryGetBySlugName {\n    PublishedCohortLibraryGetBySlugName(urlSlug: \"flu-22-to-23\") {\n        ... on Cohort {\n            id\n            name\n            urlSlug\n            conditionGroups {\n                id\n                name\n                urlSlug\n            }\n        }\n    }\n}\n",
+                "variables": ""
+              }
+            },
+            "url": {
+              "raw": "https://internal-dev-sandbox.api.service.nhs.uk/caas-published-cohort-definitions/api",
+              "protocol": "https",
+              "host": [
+                "internal-dev-sandbox",
+                "api",
+                "service",
+                "nhs",
+                "uk"
+              ],
+              "path": [
+                "caas-published-cohort-definitions",
+                "api"
+              ]
+            }
+          },
+          "response": []
+        },
+        {
+          "name": "PublishedCohortLibraryGetBySlugName",
+          "request": {
+            "auth": {
+              "type": "apikey",
+              "apikey": [
+                {
+                  "key": "value",
+                  "value": "lVlQRHlM4M111q8fnmLBe201HAWMNQJH16SU8Q4C",
```

Comment:
I guess for sandbox there will be a random string key - could you please update it to avoid confusion?

And I guess also this key can be a candidate for a variable in postman

Reply:
✅ removed as the sandbox does not have keys but it's accessible by everyone